components/scream/src/physics/p3/CMakeLists.txt

```diff
@@ -89,6 +89,9 @@ endforeach()
 add_executable(p3_tables_setup EXCLUDE_FROM_ALL p3_tables_setup.cpp)
 target_link_libraries(p3_tables_setup p3)
 
+#crusher change
```

Comment:
I didn't know about this. However, this isn't an issue since it's just the unit-testing code that is affected.

Reply:
thanks, forgot to wrap it in kokkos_enable_hip, done.